src/PackagePrivate/Doctrine/DoctrinePropertyTermStore.php

```diff
@@ -193,11 +193,11 @@ public function getTerms( PropertyId $propertyId ): Fingerprint {
 
 	private function newGetTermsStatement( PropertyId $propertyId ): Statement {
 		$sql = <<<EOT
-SELECT text, language, type_id FROM {$this->tableNames->propertyTerms()}
-INNER JOIN {$this->tableNames->termInLanguage()} ON {$this->tableNames->propertyTerms()}.term_in_lang_id = {$this->tableNames->termInLanguage()}.id
-INNER JOIN {$this->tableNames->textInLanguage()} ON {$this->tableNames->termInLanguage()}.text_in_lang_id = {$this->tableNames->textInLanguage()}.id
-INNER JOIN {$this->tableNames->text()} ON {$this->tableNames->textInLanguage()}.text_id = {$this->tableNames->text()}.id
-WHERE property_id = ?
+SELECT wbx_text, wbxl_language, wbtl_type_id FROM {$this->tableNames->propertyTerms()}
+INNER JOIN {$this->tableNames->termInLanguage()} ON {$this->tableNames->propertyTerms()}.wbpt_term_in_lang_id = {$this->tableNames->termInLanguage()}.wbtl_id
+INNER JOIN {$this->tableNames->textInLanguage()} ON {$this->tableNames->termInLanguage()}.wbtl_text_in_lang_id = {$this->tableNames->textInLanguage()}.wbxl_id
+INNER JOIN {$this->tableNames->text()} ON {$this->tableNames->textInLanguage()}.wbxl_text_id = {$this->tableNames->text()}.wbx_id
+WHERE wbpt_property_id = ?
```

Comment:
If we have the prefixes we can drop the table names. Think that is a good idea here?

Reply:
do you mean we would have only `wbpt_id` ?

Reply:
Yes

Reply:
won't work :sweat: cause we have already that column as the record id